src/nvim/testdir/test_cmdline.vim

```diff
@@ -1001,4 +1004,4 @@ func Test_read_shellcmd()
   endif
 endfunc
 
-" vim: shiftwidth=2 sts=2 expandtab	" vim: shiftwidth=2 sts=2 expandtab
+" vim: shiftwidth=2 sts=2 expandtab
```

Comment:
Weird it failed now instead of when I added this in https://github.com/neovim/neovim/commit/a576bf6196a66a0542af82c51e77e51d90a80c67#diff-9db3c00c50ebe54cfd86ad05dfaa0b9098dbca84de1c9e00fbe824e424e63353R816